plugins/copr.py

```diff
@@ -239,7 +239,7 @@ def run(self):
             copr_projectname = project[2]
             project_name = copr_username + "/" + copr_projectname
 
-        repo_filename = "{0}/_copr:{1}:{2}:{3}.repo".format(
+        self.repo_filename = "{0}/_copr:{1}:{2}:{3}.repo".format(
```

Comment:
Well, we change `repo_filename` in `_download_repo` based on the downloaded repo's ID, so I could either do it the way I do it in the commit or make `_download_repo` return the changed value or call `_runtime_deps_warning` from inside `_download_repo`. The way I did it seemed nicer to me at the time.

Reply:
I see, thank you for the explanation.

Reply:
Do we still need this change?

Reply:
Still valid question.  Much nicer would be to just call `_runtime_deps_warning` from `_download_repo` directly, or at least it could simply accept the same arguments as the `_disable_repo`.